src/App.tsx

```diff
@@ -1,203 +1,106 @@
+/* eslint-disable import/extensions */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { Header } from './Components/Header';
+import cn from 'classnames';
+import { ErrorTypes } from './types/ErrorTypes';
+import { Loader } from './Components/Loader/Loader';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [listOfTodos, setListOfTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(Status.all);
+  const [errorMessage, setErrorMessage] = useState<ErrorTypes | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    todoService
+      .getTodos()
+      .then(setListOfTodos)
+      .catch(() => {
+        setErrorMessage(ErrorTypes.UnableToLoad);
+        setTimeout(() => {
+          setErrorMessage(null);
+        }, 3000);
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  if (!todoService.USER_ID) {
     return <UserWarning />;
   }
 
+  const getFilteringTodos = listOfTodos.filter(todo => {
```

Comment:
```suggestion
  const filteredTodos = listOfTodos.filter(todo => {
```
it's already array of filtered todos
Consider using useMemo here to memoize this value